src/win/pipe.c

```diff
@@ -2200,7 +2210,7 @@ static void eof_timer_init(uv_pipe_t* pipe) {
   pipe->pipe.conn.eof_timer = (uv_timer_t*) uv__malloc(sizeof *pipe->pipe.conn.eof_timer);
 
   r = uv_timer_init(pipe->loop, pipe->pipe.conn.eof_timer);
-  assert(r == 0); /* timers can't fail */
+  assert(r == 0); (void)r; /* timers can't fail */
```

Comment:
```suggestion
  assert(r == 0);  /* timers can't fail */
  (void)r;
```
(`(void) &r;` is arguably the more idiomatic notation inside libuv but :shrug:)